modules/doc/content/help/inl/applications.md

```diff
@@ -10,10 +10,79 @@ breadth of applications continues to expand as well. The following sections high
 
 !col small=12 medium=4 large=4
-!media xfem/image21.gif style=width:100%;
+!media xfem/image21.gif style=width:100%;align:bottom;
+!row-end!
+
+<!-- Blue Crab goes here -->
+<!-- Dire Wolf goes here -->
+
+## Griffin
+
+!row!
+!col small=12 medium=8 large=8
+[Griffin Website](https://griffin-docs.hpcondemand.inl.gov/latest/)\\
+Griffin is a finite element-based reactor multiphysics application. It is suitable for steady state and time-dependent coupled neutronics calculations leveraging the various MOOSE-based thermal-fluids applications (Pronghorn, Relap-7, SAM, Sockeye, etc.) and fuel performance application (BISON). Griffin solves the linearized Boltzmann transport equation in 1D, 2D, and 3D heterogeneous and homogeneous geometries. It has been used in the analysis of pebble bed reactors (PBRs, PB-FHR), prismatic reactors (PMRs), molten-salt reactors (MSR), fast sodium-cooled reactors (FSR), microreactors, nuclear thermal propulsion (NTP), and several experimental facilities.
+
+!col small=12 medium=4 large=4
+!media application_logos/Griffin.png style=width:100%;
+!row-end!
+
+## Grizzly
+
+!row!
+!col small=12 medium=8 large=8
+[Grizzly Website](https://grizzly-docs.hpcondemand.inl.gov/latest/)\\
+Grizzly models the degradation due to normal operating conditions of nuclear power plant systems, structures, and components. The code also simulates the ability of degraded components to safely perform under a variety of conditions. Grizzly can be applied to a variety of components. However, its development focused initially on the embrittlement of reactor pressure vessels and concrete structures. Vessels can degrade and facture due to irradiation and high temperatures, while concrete can degrade due to expansive alkali-silica reactions. Grizzly has capability to model the performance effect of these and other mechanisms.
+
+!col small=12 medium=4 large=4
+!media application_logos/Grizzly.png style=width:100%;
+!row-end!
+
+## MARMOT
+
+!row!
+!col small=12 medium=8 large=8
+[MARMOT Website](https://marmot-docs.hpcondemand.inl.gov/latest/)\\
+MARMOT is a mesoscale fuel performance code. As such, it can predict the evolution of the microstructure and material properties of fuels and claddings due to stress, temperature, and irradiation damage. MARMOT can, therefore, supply microstructure-based materials models to other code that works on engineering scale, which is larger than mesoscale, with an example being BISON. MARMOT solves equations involving solid mechanics and heat conduction using the finite element method.
```

Comment:
- to other applications that work at a larger scale, such as Bison, which works at an engineering scale.